From 8e430dd7e1c5657c5ad8a74ec1acfef6cf7e8cd5 Mon Sep 17 00:00:00 2001
From: Yelena Krivosheev <yelena@marvell.com>
Date: Mon, 15 Oct 2018 12:43:53 +0300
Subject: [PATCH 0723/1051] arm64: dts: marvell: armada-37xx: add mac address
 to network interfaces

Add Ethernet aliases in the device tree, U-boot uses these aliases to
update interface's mac address in the Linux device tree.

Change-Id: Iee27c9b656e34a41b2a8dc730a22966d4a382537
Signed-off-by: Yelena Krivosheev <yelena@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60291
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index a23a3d930d35..3a131274be72 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -18,6 +18,8 @@
 	#size-cells = <2>;
 
 	aliases {
+		ethernet0 = &eth0;
+		ethernet1 = &eth1;
 		serial0 = &uart0;
 		serial1 = &uart1;
 	};
-- 
2.17.1

